src/main/java/com/example/gradecalculator/entities/GradeType.java

```diff
@@ -21,21 +21,18 @@ public class GradeType {
     @Column(precision = 5)
     private Double weightage;
 
-    @ManyToOne
-    @JoinColumn(name = "grade_system")
-    private GradeSystem gradeSystem;
 
-    public GradeType() {
-    }
-
-    public GradeType(long id, String description, GradeSystem gradeSystem) {
-        this.id = id;
-        this.description = description;
-        this.gradeSystem = gradeSystem;
-    }
 
     public String getWeightagePercentage() {
         DecimalFormat df = new DecimalFormat("#.##");
         return df.format(weightage * 100) + "%";
     }
+
+    public GradeType() {
```

Comment:
should there be an empty one?

Reply:
I need there  no-argument constructor, to  be able to instantiate the GradeType objects.